applications/FluidDynamicsApplication/custom_utilities/fluid_auxiliary_utilities.cpp

```diff
@@ -418,4 +420,73 @@ void FluidAuxiliaryUtilities::CalculateSplitConditionGeometryData<false>(
     rpModShapeFunc->ComputeNegativeExteriorFaceAreaNormals(rNormals, FaceId, GeometryData::IntegrationMethod::GI_GAUSS_2);
 }
 
+    void FluidAuxiliaryUtilities::MapEmbeddedVelocityFromSkinToVolume(
```

Comment:
```suggestion
    void FluidAuxiliaryUtilities::MapVelocityFromSkinToVolumeRBF(
```
If you want to call it map `EmbeddedVelocity` refers to the skin variable, which is `VELOCITY`. Besides I think we should make clear that RBF are used internally.

Reply:
done